python/pyspark/sql/session.py

```diff
@@ -94,12 +94,12 @@ def config(self, key=None, value=None, conf=None):
 
             >>> from pyspark.conf import SparkConf
             >>> SparkSession.builder.config(conf=SparkConf())
-            <pyspark.sql.session...
+            <pyspark.sql.session.SparkSession.Builder at 0x7fa346504c50>
```

Comment:
Hm, I don't think you want to try to match this in the doctest? that's what the ... was for, I thought

Reply:
Good call